llm-ollama.el

```diff
@@ -178,7 +220,7 @@ STREAMING if non-nil, turn on response streaming."
   ;; we really just need it for the local variables.
   (with-temp-buffer
     (let ((output (llm-request-sync-raw-output 
-                   (llm-ollama--url provider "generate")
+                   (llm-ollama--url provider (slot-value provider 'endpoint))
```

Comment:
This is fine, but stylistically the rest of the code prefers to use `(llm-ollama-endpoint provider)`, so can you change this and the other instance of this (assuming you keep the the endpoint slot?)